tty/usbacm/usbacm.c

```diff
@@ -273,6 +231,54 @@ static void usbacm_put(usbacm_dev_t *dev)
 }
 
 
+static void usbacm_handleCompletion(usb_completion_t *c, char *data, size_t len)
+{
+	usbacm_dev_t *dev = usbacm_common.devices;
+
+	dev = usbacm_getByPipe(c->pipeid);
+	if (dev == NULL)
+		return;
+
+	if (dev->flags != 0 && c->pipeid == dev->pipeBulkIN && len >= 0) {
```

Comment:
PR is approved but slight notes:
- redundant initialization for `dev`
- `len` is type `size_t` and is always `>=0`.